Functions/CatalogItems/Get-RsCatalogChildItem.ps1

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsCatalogChildItem
```

Comment:
after few discussions including @Aaron Nelson,  we settled for  Get-RsFolderContent, , although it doesn't align with the SOAP Api is explicit that what you will get is the content of a folder